### ISSUE TITLE
Oled on Pi Zero W

### ISSUE THREAD

**Issue by w1msg** [g4klx/MMDVMHost#416]
There have been recent changes to the OLED files and when I do a fresh compile Makefile.Pi.OLED it gives illegal instruction on the Pi Zero W. It does run on a Pi3. Not sure what has changed. Anyone have a solution?

**Comment by phl0** [comment]
Did you compile the binary on the target platform?

**Comment by w1msg** [comment]
Yes I did. I am guessing it has to do with all of the issues with the OLED Libraries. They havent been updated to keep up with changes in systems.

**Comment by mreckhof** [comment]
Are you still having this issue?  I just recompiled with OLED last night on a zero, and didn't have a problem.
